authui/src/authflowv2/components/label-button.css

```diff
@@ -0,0 +1,82 @@
+@layer components {
+  :root {
+    --label-btn__bg-color: transparent;
+    --label-btn__text-color: var(--color-primary-base);
+    --label-btn__border-radius: 0px;
+    --label-btn__font-family: var(--typography-label-large__font-family);
+    --label-btn__font-size: var(--typography-label-large__font-size);
+    --label-btn__line-height: var(--typography-label-large__line-height);
+    --label-btn__letter-spacing: var(--typography-label-large__letter-spacing);
+    --label-btn__font-weight: var(--typography-label-large__font-weight);
+    --label-btn__ring-width: 0px;
+    --label-btn__ring-color: transparent;
+    --label-btn__py: calc(
+      var(--label-btn__font-size) * 10 / 16
+    ); /* y padding / font size ration = 10 / 16 */
+
+    --label-btn__px: calc(
+      (var(--label-btn__py) * 2 + var(--label-btn__line-height)) / 2
+    ); /* = Height / 2 */
+
+    --label-btn__bg-color--hover: transparent;
+    --label-btn__text-color--hover: var(--color-primary-500);
+    --label-btn__ring-width--hover: var(--label-btn__ring-width);
+    --label-btn__ring-color--hover: var(--label-btn__ring-color);
+
+    --label-btn__bg-color--active: transparent;
+    --label-btn__text-color--active: var(--color-primary-500);
+    --label-btn__ring-width--active: var(--label-btn__ring-width);
+    --label-btn__ring-color--active: var(--label-btn__ring-color);
+
+    --label-btn__bg-color--disabled: transparent;
+    /* FIXME(tung): It is grey-600 with opacity in figma, confirm with designer */
```

Comment:
Are this FIXME and the FIXMEs in primary-button.css still relevant? Please remove the FIXMEs if they are resolved.

Reply:
Yes, asked designer in the meeting. Let me update it later.